File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/annotation/repeat/WrapObjects.java
package rbq.wtf.lycoris.client.wrapper.wrappers.annotation.repeat;

import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapObject;

import java.lang.annotation.*;

@Documented
@Target(ElementType.FIELD)
@Retention(RetentionPolicy.RUNTIME)
/**
 * Auto setup field
 */
public @interface WrapObjects {
    WrapObject[] value();
}

File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/annotation/repeat/WrapObjects.kt
package rbq.wtf.lycoris.client.wrapper.wrappers.annotation.repeat

import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapObject

@MustBeDocumented
@Target(AnnotationTarget.FIELD)
@Retention(AnnotationRetention.RUNTIME)
/**
 * Auto setup field
 */
annotation class WrapObjects(vararg val value: WrapObject)